src/helpers/validateSignUpEmailAddress.js

```diff
@@ -0,0 +1,13 @@
+import { ALPHANUMERIC } from "./validateEmailAddress";
+
+const validateSignUpEmailAddress = (email) => {
+    const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN
+
+    const EMAIL_DOMAIN_REGEX_STRING = emailDomain.replace(',','|').toUpperCase();
+    const EMAIL_VALIDATION_STRING =
+    `^[${ALPHANUMERIC}.!#$%&'*+/=?^_\`{|}~-]` + 
+    `+@${EMAIL_DOMAIN_REGEX_STRING}`
+    const EMAIL_VALIDATION_REGEX = new RegExp(EMAIL_VALIDATION_STRING);
+    return EMAIL_VALIDATION_REGEX.test(email.toUpperCase());
```

Comment:
Don't use `.toUpperCase()`, on line above do `new RegExp(EMAIL_VALIDATION_STRING, 'i')`
This will probably mean you can just use a-z rather than a-zA-Z

Reply:
I am not sure why my test hangs when I make these changes locally.
I am going to push it up to see if the test does not hang.

Reply:
so, the test does not hang when pushed up. I am not sure if this is only me having a hanging problem for validateSignUpEmailAddress.test.js
Something to take note.

Reply:
My test still fails locally, have to debug this issue on my end. 🙁 

So this issue is resolved.